docs/app/pages/charts/charts-sections/spark-charts/spark-chart-ng1/spark-chart-ng1.component.html

```diff
@@ -67,13 +67,21 @@
             <tr>
                 <td class="attribute">type</td>
                 <td>string</td>
-                <td>Defines the class name that is used to specify the spark line color.</td>
+                <td>Defines the class name that is used to specify the spark line and track colors.</td>
+                <td>true</td>
+            </tr>
+            <tr>
+                <td class="attribute">barColor</td>
```

Comment:
Attribute should be `bar-color`. This should reflect what the inputs are in the spark directive, not the ux-spark-ng1 component as that is only used in our docs